README.md

```diff
@@ -17,11 +17,42 @@ Analyze, Annotate, Publish. Turn documents into data.
 2. Enter the directory - `cd documentcloud`
 3. Run the dotenv initialization script - `python initialize_dotenvs.py`
    This will create files with the environment variables needed to run the development environment.
-4. Start the docker images - `inv up`
-   This will build and start all of the docker images using docker-compose. It will bind to port 80 on localhost, so you must not have anything else running on port 80. The invoke tasks specify the `local.yml` configuration file for docker-compose. If you would like to run docker-compose commands directly, set the environment variable `export COMPOSE_FILE=local.yml`.
+4. Start the docker images - `export COMPOSE_FILE=local.yml; docker-compose up`
+    -   This will build and start all of the docker images using docker-compose. It will bind to port 80 on localhost, so you must not have anything else running on port 80. The `local.yml` configuration file has the docker-compose details.
```

Comment:
I believe squarelet binds to port 80 not DocumentCloud - this also might need to be moved to after squarelet

Reply:
I forgot about Nginx at the center of the Docker squarelet network on 127.0.0.1. I think it gets a mention elsewhere. It would be good to document it explicitly here, and I'll try to. I went digging in the Dockerfiles and noticed that this custom image is used locally instead of python:3.6-alpine used on production –  do you remember why? https://hub.docker.com/r/matthewfeickert/docker-python3-ubuntu/ /  https://github.com/matthewfeickert/Docker-Python3-Ubuntu
This will slow performance on any M1 Macs since unlike the official image there is no amd64 version.

Reply:
I believe either pdfium or tesseract (or possibly both) did not run under alpine, so an ubuntu image was used.

Reply:
Maybe `3.6-bullseye` (Debian latest), `3.6-slim-bullseye` or `3.6-buster` (Debian LTS) could be used instead? It shows them at https://hub.docker.com/_/python/

Reply:
Do you use an M1 Mac?  I don't think its worth messing around with the base image if its not currently affecting anyone